setup.py

```diff
@@ -24,17 +25,26 @@ def get_version():
     raise RuntimeError('Unable to find version string in %s.' % (file_path,))
 
 
-def load_requirements(*requirements_paths):
+def load_requirements(requirements_folder):
     """
     Load all requirements from the specified requirements files.
     Returns a list of requirement strings.
     """
+    edx_platform_version = get_edx_platform_version()
+
+    if edx_platform_version == 0.9:
+        requirements_path = '{}/{}'.format(requirements_folder, 'ginkgo.in')
+    elif edx_platform_version == 0.10:
+        requirements_path = '{}/{}'.format(requirements_folder, 'hawthorn.in')
+    else:
```

Comment:
This should be an elif too, with the Ironwood version number as above. @andrey-canon

Reply:
default case

Reply:
I think the default case must be a error, since there is not a default Open edX version. @andrey-canon 
When Juniper will be released, we need to add support too add we need to add another case for that.

Reply:
imo we should revert these changes for the next edx release in order to support only one version per plugin release, in the following way.
(ginkgo, hawthorn and ironwood) version 0.5.0
(juniper) version 1.0.0
(k..) version 2.0.0

Reply:
Got it, but, we must add a changelog file, and write the version support there, so please add the changelog file. @andrey-canon